src/types/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
+type LinkItem = {
+  name: ReactNode;
+  link: string;
+  label: string;
+};
+
+export default LinkItem;
```

Comment:
please do not use default export for types or constants, you can export it like this:
export type LinkItem = ...